css/fieldmanager.css

```diff
@@ -124,6 +124,15 @@ div.fm-group-label-wrapper {
 	margin-bottom: 15px;
 }
 
+.fm-add-another-limit-reached-label {
+	/* These sizes correspond to default button sizes in core. */
+	display: inline-block;
+	font-size: 13px;
+	height: 28px;
+	line-height: 26px;
+	margin: 0 10px;
+}
```

Comment:
I wonder if we could also utilize core classes to output the desired button appearance rather than using our own class. This is especially true for `height`, `font-size`, `line-height` etc.